repository/s.json

```diff
@@ -3652,8 +3652,12 @@
 					"tags": "st3.0-"
 				},
 				{
-					"sublime_text": ">=3156",
+					"sublime_text": "3156 - 4074",
```

Comment:
```suggestion
					"sublime_text": "3156 - 4073",
```

Reply:
Haha. Wrong one. The ST4 one has to be 4075. I'll fix it.